tests/symbolicator/snapshots/SymbolicatorUnrealIntegrationTest/test_unreal_apple_crash_with_attachments.pysnap

```diff
@@ -105,145 +105,7 @@ contexts:
     seconds_since_start: 0
     time_of_crash: 636826526632420000
     type: unreal
-exception:
```

Comment:
This reverts an accidental change introduced by https://github.com/getsentry/sentry/pull/13302.
This is fine for now, as threads will still render. We need to fix this in a separate effort to improve apple crash report ingestion.